test/runnable/extra-files/mydll.di

```diff
@@ -0,0 +1,5 @@
+module mydll;
+
+export int saved_var;
```

Comment:
Good to see a DLL test in the test suite.
Are you sure that the build script and the result is actually run successfully? Don't you have to run the executable explicitly from within the shell script?
I'd be very surprised to see exported/imported variables being supported by dmd ATM (Benjamin Thaut will talk about this at DConf), especially TLS variables.